.github/workflows/author_approval.yml

```diff
@@ -0,0 +1,42 @@
+name: Author Approval Label
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  label:
+    permissions:
+        pull-requests: write
```

Comment:
Is there any narrower permission that we can give, other than `pull-requests: write`?

Reply:
I don't think there are narrower permissions: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token

Reply:
note however the token itself is only available to the final step of the workflow, which does not have any user-supplied inputs, so should be pretty secure